tests/test_api_long.py

```diff
@@ -98,35 +98,39 @@ def test_schema(self):
         self.assertTrue(ret_val)
 
     def test_schema_with_project(self):
-        """Called schema functions"""
+        """Called schema functions with project"""
+
+        project_entity = {'type': 'Project', 'id': 0}
```

Comment:
@manneohrstrom, I kept this definition outside of the `if/else`, to DRY up the `assertRaises`.